### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jbvimort**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,27 @@ A Slicer extension that enables user to interact with a Slicer scene using virtu
 
 ![](VR.png)
 
+Building steps
```
Comment (jcfr) [inline README.md]: Also add the following before the section `Building steps`:

This extension is current under development, waiting it is available in the Slicer Extension Manager, follow these instructions.

File: README.md [hunk 2]
```diff
@@ -5,6 +5,27 @@ A Slicer extension that enables user to interact with a Slicer scene using virtu
...
+--------------
+
+- Clone Slicer from https://github.com/jbvimort/Slicer/tree/VR and checkout [VR](https://github.com/jbvimort/Slicer/tree/VR) branch
```
Comment (jcfr) [inline README.md]: https://github.com/jbvimort/Slicer/tree/VR -> https://github.com/jbvimort/Slicer

Comment (jcfr) [reply README.md]: and checkout  -> and checkout  the

**Comment by jbvimort** [comment]
It should look better! :+1: 
Thanks for the feedback!

**Pull request merged**